pandas/tests/test_accent.py

```diff
@@ -0,0 +1,11 @@
+import unittest
+from pandas import read_csv, DataFrame
+from pandas.util.testing import ensure_clean
+
+class FileSystemEncodingTest(unittest.TestCase):
+    """Test compatibility with file system encoding"""
+    def test_load(self):
+        for filename in ('test_e.txt', 'test_é.txt'):
+            with ensure_clean(filename) as path:
```

Comment:
this is also not testing the issue - unicode paths should already be ok ; it is byte paths that are not working